fixed-requirements.txt

```diff
@@ -33,9 +33,8 @@ decorator==5.2.1
 oslo.config==9.6.0
 oslo.utils==7.3.0
 paramiko==3.5.1
-passlib==1.7.4
```

Comment:
Nice find!

Just a note for future reference (if anyone happens to be spelunking through old changes): passlib is required by st2-auth-backend-flat-file, so we will still pull in the requirement through that dependency. Since nothing in this repo uses passlib, it is safe to remove it from the requirements files and let pip handle resolving transitive deps like passlib.